src/resolvers-cassandra/shared.js

```diff
@@ -87,7 +87,9 @@ function parseFromToDate(fromDate, toDate) { // eslint-disable-line no-unused-va
 }
 
 function parseLimit(limit) {
-  return limit > 0 ? limit : 15;
+  const DEFAULT_LIMIT = 15;
+
+  return limit && limit > 0 ? limit : DEFAULT_LIMIT;
```

Comment:
This can just be `limit > 0` since undefined and null are non-positive:

![image](https://user-images.githubusercontent.com/1086421/29584216-e8297ff8-8737-11e7-8609-47b189811f9f.png)